[Project] OmegaStation 2.0

Lynameep opened the pull request:
### Intent of your Pull Request

Lyna's attempt at the bounty submitted by Altoids1 #6034.

This map has quite a few issues but the things requested in the bounty should be done. Feel free to throw any requests in here and I shall work on them.

#### Changelog

:cl:  
wip: very Omegastation
experimental: much experimental
/:cl:

Lynameep committed:
[Project] OmegaStation 2.0

our hearts sing in the chaos

Lynameep committed:
map mergiffy me daddy

Lynameep committed:
Bug Fixs and Additions

Moved HoP office around

Added Plasma glass to the south part of the engine room

Two whole emitters instead of one

Engineering/atmos has been moved two tiles West.

Security now has a security vendor and bomb squad locker

Gravity Gen now has a bit more room and a scrubber

Paramedic, Mining Medic, and Signal Tech are available as jobs now.

Lynameep committed:
Missed a spot

o o p s

Altoids1 reviewed (changes requested):
I'm not really a fan of what you ended up doing to Engineering, to be honest.

![image](https://user-images.githubusercontent.com/29939414/60761116-a5e56400-a007-11e9-856a-54b50a79a50c.png)

I get the reinforced walls are important and in-line with what I requested, but I would still like some pair of airlocks over where the old entrance was, perhaps in a configuration similar to the one between Genetics and RnD on Boxstation. Having the engineers be forced to wander around and into atmos in order to get to the engine feels a bit silly, especially since some key equipment is stored in the engineering room that now lacks immediate access.

Altoids1 reviewed (dismissed):
![bad_tile](https://user-images.githubusercontent.com/29939414/60761144-3328b880-a008-11e9-89f6-d0609569d18d.png)

Throughout the station are these weird poorly-rotated caution tiles, on basically every entrance that has them.

Lynameep committed:
Update omegastation.dmm

+ Implemented fail-safe spawns for disabled jobs
+ Fixed a few misplaced caution corner bit thingies
+ Added a double airlock entrance from engineering to the engine
+ Added a tiny secret room to layout my frustrations with the numerous tiny issues with this map that actually didn't have much impact on gameplay

Lynameep committed:
Operation Ktlw Fix

Fixed everything outlined in ktlw's comment as well as even MORE misplaced corner bits.

Lynameep committed:
Fixes and stuff

Added an additional Miner spawn/locker and gave them a bit more room.
Moved Engineer vendor out of the engine
Gave Tcoms their very own clothing locker. Wow!

Lynameep committed:
Tcomms Wire Fix

Also added an evac sign pointing to evac.

Youre welcome!

Lynameep committed:
Job Issue Fix

thx nich

Amelia0010 committed:
Update job_changes.dm
